src/util/index.js

```diff
@@ -437,3 +437,19 @@ export const sanitizeIdAttr = str => {
 		.trim()
 		.toLowerCase()
 }
+
+/**
+ * Extract the hexadecimal value of the color from a non-stackable color.
+ *
+ * @param {string} value Non-stackable color
+ * @return {string} Hexadecimal value of the color
+ */
+export const extractColor = value => {
+	const color = value
```

Comment:
This line doesn't really do anything, you can just use `value` instead of `color` in the rest of the function